docs/glossary.md

```diff
@@ -183,10 +183,14 @@ An append-only log of Events that represents the full state a Workflow Execution
 
 _Tags: [term](/tags/term), [explanation](/tags/explanation)_
 
+#### [Failback](/cloud/multi-region)
+
+After Temporal Cloud has resolved an outage or incident involving a failover, a failback process shifts Workflow Execution processing back to the original Temporal Namespace that was active before the incident.
```

Comment:
Small distinction here (my mistake I think I drafted this text). We should probably say "shifts Workflow Execution processing back to the original region that was active before the incident." because it's still the same Namespace, just the region is different.

Reply:
But I realize we have defined Failover similarly as well -- so this is more of a nit. Feel free to ignore if this increases the scope of the change too much.

Reply:
Fixing both.